src/App.js

```diff
@@ -1,14 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
+// const testData = {
+//   id: 1,
+//   sender: 'Vladimir',
+//   body: 'why are you arguing with me',
+//   timeStamp: '2018-05-29T22:49:06+00:00',
+//   liked: false,
+// };
+// const testData = {
+//   id: 1,
+//   sender: 'Vladimir',
+//   body: 'why are you arguing with me',
+//   timeStamp: '2021-05-15T22:49:06+00:00',
+//   liked: false,
+// };
+// const testData = [
+//   {
+//     id: 1,
+//     sender: 'Vladimir',
+//     body: 'why are you arguing with me',
+//     timeStamp: '2021-05-15T22:49:06+00:00',
+//     liked: false,
+//   },
+// ];
+
+// const testConvo = chatMessages;
```

Comment:
Style: It's industry practice to clean up commented out code before creating a PR.